pkg/ebpf/c/tracee.bpf.c

```diff
@@ -783,10 +783,16 @@ typedef struct task_info {
     u8 container_state;   // the state of the container the task resides in
 } task_info_t;
 
+typedef struct binary {
+    u32 mnt_id;
+    char path[MAX_BIN_PATH_SIZE];
+} binary_t;
+
 typedef struct proc_info {
     bool new_proc; // set if this process was started after tracee. Used with new_pid filter
     bool follow;   // set if this process was traced before. Used with the follow filter
-    char binary[MAX_BIN_PATH_SIZE];
+    struct binary binary;
+    u32 binary_no_mnt; // used in binary lookup when we don't care about mount ns. always 0.
```

Comment:
this one.

Reply:
Is this meant to be set when the binary filter ignores the mount namespace?

Reply:
No. It is meant to be always 0.